tests/conn_can/Makefile

```diff
@@ -7,10 +7,6 @@ BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h nucleo32-f031 nucleo32-f04
                              nucleo-l053 stm32f0discovery telosb wsn430-v1_3b \
                              wsn430-v1_4 z1
 
-
-CFLAGS += -DDEVELHELP
-CFLAGS += -DLOG_LEVEL=LOG_ALL
```

Comment:
Why change the log level?

Reply:
unrelated -yes - but having `LOG_ALL` is inconsistent with all other tests where it is not enabled. So why here, and if we want to have it then it should be set for all tests (and then use Makefile.test_common), or not?

Reply:
@smlng - because it is another thing to test for. If you keep that line, I can look at the diff and say "I acknowledge that this PR enables DEVELHELP for all tests except minimal". If you remove it, I have to dig through the test to ensure it makes no difference...

I agree having "LOG_ALL" enabled in that test, without comment, is weird. It's just unrelated.

Reply:
okay, so I leave `CFLAGS += -DLOG_LEVEL=LOG_ALL` as is - and we may remove it in a separate PR, okay?